components/actionbutton/stories/actionbutton.test.js

```diff
@@ -7,17 +7,15 @@ export const ActionButtons = (args, context) => {
 		${Template(args, context)}
 		${Template({
 			...args,
-			iconName: undefined,
 		}, context)}
 		${Template({
 			...args,
 			hideLabel: true,
 		}, context)}
 		${Template({
 			...args,
-			hasPopup: "menu",
-			label: "Has pop-up",
-			iconName: undefined,
+			hasPopup: "true",
+			hideLabel: true,
 		}, context)}
```

Comment:
Do you think it's worth adding a fifth action button here that displays like the original, with label text + the hold icon? Covering the hold icon for both scenarios could be useful.

Reply:
@aramos-adobe I think we just need to update `ActionButtons` in the test file to have the fifth button like we have in `ActionButtonsWithIconOptions`.

Docs page ✅
<img width="979" alt="Screenshot 2024-09-23 at 10 25 17 AM" src="https://github.com/user-attachments/assets/fe2f3daf-58c0-4ad0-b319-189c1bd2ba74">

Test view
<img width="1334" alt="Screenshot 2024-09-23 at 10 25 03 AM" src="https://github.com/user-attachments/assets/a0c3338b-9143-45cf-9c10-4a24791926f6">